config.json

```diff
@@ -263,6 +263,14 @@
         "prerequisites": [],
         "difficulty": 3
       },
+      {
+        "slug": "transpose",
+        "name": "Transpose",
+        "uuid": "d41e71c8-1b2d-4ea7-a3c4-def79b5827d0",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
+      },
```

Comment:
Could you update the difficulty and ordering?

Reply:
I meant it to be a 3, so it's at the right place, I just forgot to update the difficulty, why do I keep forgetting, sorry about that!